src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -6,30 +6,45 @@
 
 import com.revrobotics.CANSparkMax;
 import com.revrobotics.SparkAbsoluteEncoder;
+import com.revrobotics.CANSparkBase.IdleMode;
 import com.revrobotics.CANSparkLowLevel.MotorType;
+
+import edu.wpi.first.wpilibj.DigitalInput;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants.HopperConstants;
 import frc.robot.Ports;
+import frc.robot.Ports.HopperPorts;
 
 public class Intake extends SubsystemBase {
-  /** Creates a new Intake. */
+  //intake
   // private static CANSparkMax rotationNEO;
   private static CANSparkMax rollerNEO;
```

Comment:
can rename to "intakeMotor" instead of "roller"

Reply:
use "rename symbol" option in vscode to change all occurrences of this variable!